bitnami/vault/values.yaml

```
@@ -1760,6 +1760,17 @@ injector:
     ## @param injector.serviceAccount.automountServiceAccountToken Automount service account token for the server service account
     ##
     automountServiceAccountToken: false
+
+  webhook: {}
+  ## @param injector.webhook Enabling specify which namespace you want to work with webhook
+  ## Example:
+  ##   namespaceSelector:
+  ##     matchExpressions:
+  ##     - key: kubernetes.io/metadata.name
+  ##       operator: In
+  ##       values:
+  ##         - "{{ .Release.Namespace }}"
+  #namespaceSelector: {}
```

I will apply this small change to help improve the values.yaml readability.
```suggestion
  ## @param injector.webhook.namespaceSelector Enabling specify which namespace you want to work with webhook
  ## Example:
  ##   namespaceSelector:
  ##     matchExpressions:
  ##     - key: kubernetes.io/metadata.name
  ##       operator: In
  ##       values:
  ##         - "{{ .Release.Namespace }}"
  webhook:
    namespaceSelector: {}
```